### DISCUSSION TITLE
How to correctly access the uuid attribute of a SecurityRule from Panorama?

### DISCUSSION THREAD

**Discussion by johnjtorres** [PaloAltoNetworks/pan-os-python#386] [Q&A]
What is the correct way to access the uuid attribute from a SecurityRule that was pulled from Panorama? All of my rules show that the uuid is None. 

```python
pan = Panorama(hostname=hostname, api_key=api_key)
device_group = pan.add(DeviceGroup("MyDeviceGroup"))

post_rulebase = device_group.add(PostRulebase())
rules = SecurityRule.refreshall(post_rulebase)
for rule in rules:
    print(rule.name, rule.uuid)
```
```
# Output
rule1 None
rule2 None
rule3 None
rule4 None

# Expected output
rule1 2f563aaa-d228-4f93-9f8e-fd898d62178d
rule2 06199882-24e2-4c34-9f90-ba6783ca9065
rule3 955057fc-686c-46ee-94cd-3e962d063a2c
rule4 3dd11297-112a-410b-9627-c0aeb2b2026e
```

Not sure if I am doing something wrong or if I need to submit an issue.

DEBUG4:pan.xapi:Python version: 3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)]
DEBUG4:pan.xapi:xml.etree.ElementTree version: 1.3.0
DEBUG4:pan.xapi:ssl: OpenSSL 1.1.1d  10 Sep 2019
DEBUG4:pan.xapi:pan-python version: 0.16.0

Panorama version 10.0.8
pan-os-python==1.6.0

**Comment by shinmog** [comment]
This should be fixed by #395
